### ISSUE TITLE
Reproduce results on DarkZurich val set

### ISSUE THREAD

**Issue by DeppMeng** [W-zx-Y/DANNet#1]
Hi,

I am trying to reproduce the reported result on DarkZurich validation set: 36.76.
I used your codes and the same default training configs, and trained the model on a single Tesla V100 GPU with PyTorch 1.7.0. The validation miou is 34.86. Is this result in a reasonable variance range?

**Comment by DeppMeng** [comment]
Update:

I visualized the images output by lightnet, it seems that the lightnet is not properly optimized.

Here are the original image, enhanced image and light image output by **the trained model given in this repo**:
![image](https://user-images.githubusercontent.com/26196079/118779077-4a4fb700-b8bd-11eb-8754-652cfd6b47ca.png)
![image](https://user-images.githubusercontent.com/26196079/118779167-5e93b400-b8bd-11eb-9eec-fc32a63ee92e.png)
![image](https://user-images.githubusercontent.com/26196079/118779230-6d7a6680-b8bd-11eb-800a-db1f13be590b.png)

Here are the original image, enhanced image and light image output by **my reproduced model**:
![image](https://user-images.githubusercontent.com/26196079/118779077-4a4fb700-b8bd-11eb-8754-652cfd6b47ca.png)
![image](https://user-images.githubusercontent.com/26196079/118780249-6dc73180-b8be-11eb-9bb9-94e2cdb438b3.png)
![image](https://user-images.githubusercontent.com/26196079/118780289-77509980-b8be-11eb-8dda-82fcf15d6d39.png)

Is there any suggestions for this issue?

Thanks!

**Comment by W-zx-Y** [comment]
Hi @DeppMeng,
Actually, we didn't encounter this problem before. What about the performance on the test set?

**Comment by DeppMeng** [comment]
I did not try to submit a test. Thank you for your reply, I am trying to locate the problem.

**Comment by DeppMeng** [comment]
Hi @W-zx-Y ,

Can you provide a training log of the model which is very helpful for me debugging. Thanks!

**Comment by W-zx-Y** [comment]
Sorry for our late reply. We didn't save the training log into a file during training. Have you fixed the problem in the relighting part?

**Comment by DeppMeng** [comment]
Yes, I re-ran the experiment again and the re-light network output is similiar to the enhanced image you provided. Don't know what causes the lightnet fail to converge in first trial. Maybe some instability in training?

**Comment by creater-zq** [comment]
> Hi,
> 
> I am trying to reproduce the reported result on DarkZurich validation set: 36.76.
> I used your codes and the same default training configs, and trained the model on a single Tesla V100 GPU with PyTorch 1.7.0. The validation miou is 34.86. Is this result in a reasonable variance range?
I had a similar problem,The validation miou is 34.38。

**Comment by creater-zq** [comment]
> Hi,
> 
> I am trying to reproduce the reported result on DarkZurich validation set: 36.76.
> I used your codes and the same default training configs, and trained the model on a single Tesla V100 GPU with PyTorch 1.7.0. The validation miou is 34.86. Is this result in a reasonable variance range?

I had a similar problem,the validation miou is 34.86。

**Comment by Jumponthemoon** [comment]
> Yes, I re-ran the experiment again and the re-light network output is similiar to the enhanced image you provided. Don't know what causes the lightnet fail to converge in first trial. Maybe some instability in training?

@DeppMeng Hi, is your experiments results is exactly the same with the authors? My lighting effect is align with authors' in general but seems the color is tend to be green rather than blue.

**Comment by Urbenyy** [comment]
Mine is pink
![GOPR0356_frame_000321_rgb_anon_light](https://github.com/W-zx-Y/DANNet/assets/96567010/5298078f-fd88-43e6-b89e-b2d2e5f1f860)
![GOPR0356_frame_000321_rgb_anon_enhancement](https://github.com/W-zx-Y/DANNet/assets/96567010/1eba5d28-f93e-4fd5-b426-a1219acaf04d)


> > Yes, I re-ran the experiment again and the re-light network output is similiar to the enhanced image you provided. Don't know what causes the lightnet fail to converge in first trial. Maybe some instability in training?
> 
> @DeppMeng Hi, is your experiments results is exactly the same with the authors? My lighting effect is align with authors' in general but seems the color is tend to be green rather than blue.
